drivers/net/phy/realtek.c

```diff
@@ -1249,6 +1406,16 @@ static struct phy_driver realtek_drvs[] = {
 		.read_page      = rtl821x_read_page,
 		.write_page     = rtl821x_write_page,
 		.soft_reset     = genphy_soft_reset,
+	}, {
+		.match_phy_device = rtl8221_vb_cg_c45_match_phy_device,
+		.name           = "RTL8221B-VB-CG 2.5Gbps PHY (C45)",
+		.config_init	= rtl822x_c45_config_init,
+		.get_rate_matching = rtl822x_get_rate_matching,
+		.get_features   = rtl822x_c45_get_features,
+		.config_aneg    = rtl822x_c45_config_aneg,
+		.read_status    = rtl822x_c45_read_status,
+		.suspend        = rtl822x_c45_suspend,
```

Comment:
Why not assign `genphy_c45_pma_suspend` directly here?

Reply:
I did have some code here, and used it for debugging messages, but now indeed it is empty and I planned to remove , if it would stay like this. So agree, I only need to use it once to check the order of execution .config_init and ,resume, to see if I need to call genphy_c45_pma_resume() from rtl822x_c45_config_init().